plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -738,9 +741,26 @@ export class DeferredPersonOverrideWriter {
             undefined,
             'pendingPersonOverride'
         )
-
+        deferredPersonOverridesWrittenCounter.inc()
         return []
     }
+}
+
+const deferredPersonOverridesProcessedCounter = new Counter({
+    name: 'deferred_person_overrides_processed',
+    help: 'Number of pending person overrides that have been successfully processed',
+})
+
+export class DeferredPersonOverrideWorker {
+    // The advisory lock identifier/key used to ensure that only one process is
+    // updating the overrides at a time.
+    public readonly lockId = 567
+
+    private writer: PersonOverrideWriter
+
+    constructor(private postgres: PostgresRouter, private kafkaProducer: KafkaProducerWrapper) {
+        this.writer = new PersonOverrideWriter(this.postgres)
```

Comment:
Next step is going to be making this `PersonOverrideWriter` able to be substituted with one that doesn't require the mapping table and constraints, and preparing the process for moving data from the existing table to the new one.